src/test/java/com/jadajabajava/services/EmployerServiceTest.java

```diff
@@ -0,0 +1,63 @@
+package com.jadajabajava.services;
+
+import com.jadajabajava.entities.Employer;
+import com.jadajabajava.repositories.EmployerRepository;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class EmployerServiceTest {
+
+    @Mock
+    private EmployerRepository employerRepository;
+
+    @InjectMocks
+    private EmployerService employerService;
+
+    private static Employer existingEmployer;
+
+    @BeforeAll
+    static void setUp() {
+        Long existingEmployerId = 3L;
+        Long existingEmployerRemoteId = 12L;
+        String existingEmployerTitle = "Cowabunga LLC";
+
+        existingEmployer = Employer
+                .builder()
+                .id(existingEmployerId)
+                .remoteId(existingEmployerRemoteId)
+                .title(existingEmployerTitle)
+                .build();
+    }
+
+    @Test
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/jada-jaba-java/job-seeker-service/pullRequest?prid=5720132)